src/utils/daos.c

```diff
@@ -1317,6 +1341,9 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 			"	--path=PATHSTR     container namespace path\n"
 			"container create common optional options:\n"
 			"	--type=CTYPESTR    container type (HDF5, POSIX)\n"
+			"	--mode=FLAG        In case of POSIX type, select consistency mode:\n"
+			"                          relaxed: weaker consistency semantics.\n"
+			"                          balanced (default): stronger consistency semantics.\n"
```

Comment:
Minor: In my terminal, this displays as:
```
        --mode=FLAG        In case of POSIX type, select consistency mode:
                          relaxed: weaker consistency semantics.
                          balanced (default): stronger consistency semantics.
```
I think for the second and third lines, we also needs tabs instead of spaces. (at least for the part before --mode)